### ISSUE TITLE
Barely functioning in iOS. Perfect in Android. QR recognition script or app?

### ISSUE THREAD

**Issue by grup40** [mebjas/html5-qrcode#893]
**Describe the bug**
- Scanning efficiency is very poor on iOS. but works perfectly well on Andoird. 
In fact, the android run app will scan a code through the screen on the iOS device and detect it, where the iOS device struggles to detect the code.

**Describe the browser:**
 - OS: [iOS & Android]
 - Browser [iOS: Safari and Chrome tested. Andoird: Chrome
 - Version [iOS 17. Android 14]

**Screenshots**

**Additional context**
None other than above. Works PERFECTLY on android, every single time.
iOS barely works.

Is this something to do with the QR recognition scrip running locally or something app based?

Thanks in advance, I know nothing about coding so be kind! :D

**Comment by ROBERT-MCDOWELL** [comment]
don't open an issue where all the inherent problems you have is coming from your device.
also check the closed issue that many many issues talked about IOS and focus/zoom problem.
contact apple and IOS developers to solve your problems as if html5-qrcode is DEPEDENT of JAVASCRIPT engine and cannot
go beyond its sandbox

**Comment by grup40** [comment]
Thanks for your rather harsh reply.
As I said I am not in anyway a software engineer. Was only asking for help.

I'm not sure how it's inherently the device. Because it appears to work on
multiple operating systems apart from iOS. The result of using it via an
iOS device is constant. Various iOS devices using varying browsers. All are
ineffective.
Android and windows work flawlessly on multiple devices.

**Comment by alingeopress** [comment]
Don't listen to this guy, he is answering the same thing to every issue, he never wrote a line of code in his entire life.

About the issue, I am facing the same problem but for both devices. The library itself it's not really great for production uses atm.

**Comment by grup40** [comment]
That's rich 😅

**Comment by ROBERT-MCDOWELL** [comment]
do you really think you'll be helped reacting like that to the developers here? where do you think you are? on tiktok? you are going to be blocked if you continue.

**Comment by grup40** [comment]
Look...I came on here and asked a genuine question.
I had a rather unkind and unhelpful reply.

Happy to apologise.

Would you be happy to help with the issue as originally discussed?

Thanks

**Comment by ROBERT-MCDOWELL** [comment]
listen guy who registered on github 2 days ago only, if you didn't read the code of conduct of this project you should read it and stop to bully  me and others. github is not a fancy place where you can do everything for your selfish needs. rather you should learn the rule and learn how to use github or you will be banned very quickly. I answered you with a genuine solution, but you prefer to be the king to be served? without respect no help from anyone here. If you are really stuck contact the main author and pay him to solve your solution.

**Comment by ROBERT-MCDOWELL** [comment]
and if you think my answer is not useful so I copy and paste it again as apparently you still didn't read it
don't open an issue where all the inherent problems you have is coming from your device.
also check the closed issue that many many issues talked about IOS and focus/zoom problem.
contact apple and IOS developers to solve your problems as if html5-qrcode is DEPENDENT of JAVASCRIPT engine and cannot
go beyond its sandbox. if you are not a developer and don't know anything about language just hire someone here to solve your problem.

**Comment by grup40** [comment]
I am not bullying you.

I asked for help and your reply was unhelpful and unkind in it's wording.

Sorry for stating that.

**Comment by grup40** [comment]
I read it and it did not answer my question.
Hence the additional information given in my reply.

**Comment by ROBERT-MCDOWELL** [comment]
Search on closed issues tracker "IOS zoom focus" and you will solve it.

**Comment by Sukyzaur** [comment]
Yeah as @alingeopress stated before, this guy is just random talking on here he calls himself "developer expert in Perl, Actionscript & HTML" this guy is a joke lmfao

**Comment by Sukyzaur** [comment]
@ROBERT-MCDOWELL get a job man

**Comment by ROBERT-MCDOWELL** [comment]
@Sukyzaur great developer you are...  a nice piec of sh..t

**Comment by diegobustosk** [comment]
I am Having the same Issue.
